src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
Yep this was my thinking as well, but I had already created the folders before I decided to look into those files and see what was going on ;) I'll fix it up